libraries/canjs/src/components.js

```diff
@@ -143,21 +151,21 @@ Component.extend({
     </div>
   `),
   ViewModel: DefineMap.extend({
+    seal: false// webcomponentsjs/shadydom sets a property when addEventListener is called
+  }, {
     eventHandled: {type: 'boolean', value: false},
-    handleTestEvent: function() {
+    handleEvent: function() {
       this.eventHandled = true;
     }
   }),
   events: {
     inserted: function() {
       let wc = this.element.querySelector('#wc');
-      wc.addEventListener('camelEvent',
-        this.viewModel.handleTestEvent.bind(this.viewModel));
+      wc.addEventListener('camelEvent', this.viewModel);
```

Comment:
this should be changed back to `handleTestEvent`. That's what the other tests are using now.

Reply:
Ah ok, so you explicitly don’t want to use [EventListener’s handleEvent](https://developer.mozilla.org/en-US/docs/Web/API/EventListener#handleEvent())?

Reply:
it doesn't matter what they use, `handleEvent` is standard. If your `viewModel` has such method, it's by standards good to be set as listener, which is nothing different than a bind, except you don't need to bind and you can remove it later on whenever you want.

TL;DR it makes no sense to say _don't use handleEvent_ when it's the platform providing such mechanism, right?

Reply:
I didn't want to use handleEvent because, while it is a standard, it's one that I think most people are unfamiliar with and I didn't want to have to go out of our way to explain it in comments in all of the tests. I'd prefer to stick with the named handler because it's just one less thing to have to deal with.

Reply:
if you want to remove that listener later on, it's 2 things that you have to deal.

Readdress that method as bound one and use it for both `addEventListener` and `removeEventListener`.

I don't think it should be the scope of this repository to limit `addEventListener` standard behavior, neither the one that should prevent developers from learning ... quite the opposite, to be honest ... but if it makes your life easier as maintainer of this CanJS part, which I am not sure myself what is this about, then it's OK, your lib, your rules.

Reply:
here a link just in case somebody reads and would like to know what is this about:
https://dom.spec.whatwg.org/#callbackdef-eventlistener